stonesoup/sensor/radar/radar.py

```diff
@@ -62,6 +68,12 @@ def measure(self, ground_truths: Set[GroundTruthState], noise: Union[np.ndarray,
                                       timestamp=truth.timestamp,
                                       groundtruth_path=truth)
             detections.add(detection)
+        
+        # Generate clutter at this time step
+        if self.clutter_model is not None: 
+            self.clutter_model.measurement_model = measurement_model
```

Comment:
Could do this in constructor (`__init__`) (and on line 231)

Reply:
Good idea. Unfortunately, the sensor's measurement model isn't defined until its `measure()` function gets called. I think this was done so that the measurement model would hold the correct position and rotation offset at each time step. However, this means that the `ClutterModel.measurement_model` attribute can't be set in it's own constructor or in the radar sensor's constructor. We might actually want the measurement model to change at each time step for the same reasons as the radar's one changes. 

The only way I see around this would be to move the definition of the measurement model to a constructor for the radar. However this might mess up the measurement model for the reasons described above.

Reply:
Very good point. Its good as is.